Updated Readme

anurag8546 opened the pull request:
Added new features to Readme.md

johnwalicki reviewed (changes requested):
Thanks for the contribution.  This looks great.  Can you make 2 changes before we merge this.

- Remove the unnecessary `<br>` lines from your commit.
- Remove one of the trailing newlines from your commit.

Thanks so much for joining the Open Horizon community.

johnwalicki commented:
Some of the commits are messy multiple edits to links.  Maybe we could recommend to squash this into one commit ?

johnwalicki commented:
@anurag8546  It would be great to merge your contribution into the Open Horizon project. See above for some requested changes. Can you git squash these commits and sign the DCO?

anurag8546 commented:
@joewxboy  I've done both the required changes -
1) Removed unnecessary <br> from code
2) Removed  the trailing newline from the commit.

This still gives all checks have failed.

anurag8546 commented:
> @joewxboy I've done both the required changes -
> 
> 1. Removed unnecessary  from code
> 2. Removed  the trailing newline from the commit.
> 
> This still gives all checks have failed.

@johnwalicki

joewxboy commented on profile/README.md:
```diff
@@ -10,7 +10,7 @@
 <img src="https://img.shields.io/github/stars/open-horizon?label=Stars" alt="Total Stars">
 <!--
-<a href="https://twitter.com/lf_edge)" alt="twitter"><img src="https://img.shields.io/badge/twitter-@lf_edge--twitter-yellow.svg" alt="twitter"></img></a>
+<a href="https://twitter.com/lf_edge" alt="twitter"><img src="https://img.shields.io/badge/twitter-@lf_edge--twitter-yellow.svg" alt="twitter"></img></a>
```
Please remove closing `</img>` tag.  Image is not an HTML container, so it does not have a closing tag in HTML 4.x and HTML 5.

joewxboy commented:
@anurag8546 When you click on each individual commit, you are shown details about which checks are failing for that specific commit, and potentially how to resolve them.  Here's one:
https://github.com/open-horizon/.github/pull/17/checks

Let me know if the description on how to sign the commits is unclear or hard to follow.

joewxboy commented:
@anurag8546 Are you available to amend your commits to add a sign-off?  That will allow this PR to be merged.  Thanks.

joewxboy commented:
No DCO, closing
